src/clients/cassandra/CassandraConnector.js

```diff
@@ -93,6 +93,30 @@ function executeQuery(query, params, options) {
 }
 
 /**
+ * @param {string} query
+ * @param {string[]} params
+ * @param {int} fetchSize
+ * @param {string} pageState
+ * @returns {Promise.<object[]>}
+ */
+function executeQueryWithPageState(query, params, pageState, fetchSize) {
+  return new Promise((resolve, reject) => {
+    const DEFAULT_FETCH = 15;
+
+    let options = {
+      fetchSize: fetchSize || DEFAULT_FETCH
```

Comment:
This may be better as fetchSize > 0 ? fetchSize : DEFAULT_FETCH since it'll handle the degenerate case where someone calls this with a negative fetch size.